app/jiraapi.py

```diff
@@ -175,6 +175,11 @@ def __getattr__(self, name):
         return partial(self.http, name)
 
     def create_issue(self, project, summary, description, issuetype_name, **kwargs):
+        MAX_LENGTH = 30000
+
+        if description and len(description) > MAX_LENGTH:
```

Comment:
Perhaps insert a warning into the text that it was too long and has been truncated?